tests/whitelist.txt

```diff
@@ -4,9 +4,6 @@ integration.fileserver.test_fileclient
 integration.fileserver.test_roots
 integration.loader.test_ext_grains
 integration.loader.test_ext_modules
-integration.loader.test_globals
-integration.loader.test_interfaces
-integration.loader.test_loader
```

Comment:
I tried adding unit tests to the whitelist, but that didn't work.... Do I need to have the test suite run another command to get it to run the unit tests?